lesson08/assignment/vue-router-01/src/components/photos.vue

```diff
@@ -0,0 +1,34 @@
+<script>
+export default {
+  name: 'photos',
+  props: ['image']
```

Comment:
Make sure you are validating your props!

Reply:
You are taking in an object, that then expects several values: src, all, caption. you need to be validating that your component receives props in this form. 
https://vuejs.org/v2/guide/components-props.html#Passing-the-Properties-of-an-Object